src/app/pages/bajas/form-solicitud/form-solicitud.component.ts

```diff
@@ -0,0 +1,500 @@
+import { Component, OnInit, Output, EventEmitter, ViewChild, Input } from '@angular/core';
+import { combineLatest, Observable } from 'rxjs';
+import { FormBuilder, FormGroup, Validators, ValidatorFn, AbstractControl, ValidationErrors, FormArray } from '@angular/forms';
+import { TranslateService, LangChangeEvent } from '@ngx-translate/core';
+import { debounceTime, distinctUntilChanged, map, startWith, switchMap } from 'rxjs/operators';
+import { MatPaginator, MatTableDataSource } from '@angular/material';
+import { BajasHelper } from '../../../helpers/bajas/bajasHelper';
+import { MovimientosHelper } from '../../../helpers/movimientos/movimientosHelper';
+import { FormatoTipoMovimiento, Movimiento } from '../../../@core/data/models/entrada/entrada';
+import { DomSanitizer, SafeResourceUrl } from '@angular/platform-browser';
+import { PopUpManager } from '../../../managers/popUpManager';
+import { UserService } from '../../../@core/data/users.service';
+import { NuxeoService } from '../../../@core/utils/nuxeo.service';
+import { HttpErrorResponse } from '@angular/common/http';
+import { DocumentoService } from '../../../@core/data/documento.service';
+@Component({
```

Comment:
(En general) Dejar las constantes en un mismo lado, o por lo menos al comienzo del archivo, antes de la declaración del componente. Por ejemplo para este archivo:

```suggestion
import { DocumentoService } from '../../../@core/data/documento.service';

const SIZE_SOPORTE = 5 * 1024000;

@Component({
```
Y ajustar las L27, L421 y L428 según sea adecuado

Lo anterior con el fin de
- Que las constantes queden en un mismo lado
- Que cuando se tengan que traer, bien sea de configuración API, de parametros API o de otro lado, "solo sea cambiarlas en un mismo lado"